public/js/selfoss-base.js

```diff
@@ -20,13 +20,18 @@ var selfoss = {
         source: '',
         ajax: true
     },
-
+    
```

Comment:
You're telling me that despite the fact that Selfoss has no coding guidelines and this is inconsistent over the whole codebase. I think SSilence focuses on more important things ;)